Xamarin.Forms.Controls/Xamarin.Forms.Controls.csproj

```diff
@@ -34,7 +35,9 @@
     <EmbeddedResource Include="GalleryPages\crimson.jpg" />
     <EmbeddedResource Include="coffee.png" />
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="System.Threading.Thread" Version="4.3.0" />
     <PackageReference Include="Xam.Plugin.DeviceInfo" Version="3.0.2" />
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.0.6" Condition="$(TargetFramework) == 'uap10.0.14393'" />
```

Comment:
Is this reference necessary from the shared .NET library (Xamarin.Forms.Controls)?